rover/source/index.rst

```diff
@@ -1,17 +1,14 @@
 .. _home:
 
-================
-ArduPilot: Rover
-================
+===============
+ArduPilot Rover
+===============
 
 ..  raw:: html
  
     <p style="text-align:center;color:red;"><strong>
-    Success with ArduPilot requires that the 'First Time Setup' and 'First Drive and Tuning' sections of this documentation be read and followed!</strong>
-    
-.. tip::
-
-   Keep up with the latest ArduPilot related blogs on `ArduPilot.org! <https://ardupilot.org/>`__
+    Success with Rover requires that the 'First Time Setup' and 'First Drive and Tuning' sections of this documentation be read and followed!</strong>
+    <br></br>
```

Comment:
doesn't existing spacing suffice?

Reply:
Without this the red text is very close to the pictures on the home pages. This seemed to make the red text and the image flow better.

Reply:
okay